spec/models/user_spec.rb

```diff
@@ -177,19 +178,55 @@
       end
     end
 
-    describe "for inactive users" do
-      subject{ described_class.new active: false }
+    describe 'with an existing technical user' do
+      subject do
+        organization = create(:organization)
+        create(:technical_user, name: 'tech', organization: organization)
+        second_tech_user = build(:technical_user, name: 'tech', organization: organization)
+        second_tech_user.memberships << build(:membership, user: second_tech_user, organization: organization)
+        second_tech_user
+      end
+      it(nil, probedock: { key: 'iwzr' }) do
+        expect(subject).not_to be_valid
+        expect(subject.errors[:name].size).to eq(1)
+        expect(subject.errors[:name][0]).to eq('must be unique in organization')
+      end
```

Comment:
Add a test to ensure that a technical user can be created in an organization with a human member who has the same name.